x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/cti/threat_match_row.tsx

```diff
@@ -46,7 +47,7 @@ export const ThreatMatchRow = ({
     contextId,
     eventId,
     indicatorReference: get(data, REFERENCE)[0] as string | undefined,
-    indicatorProvider: get(data, PROVIDER)[0] as string | undefined,
+    indicatorProvider: get(data, FEED_NAME)[0] as string | undefined,
```

Comment:
If we're changing the semantic meaning of this prop we should change its name as well.
```suggestion
    feedName: get(data, FEED_NAME)[0] as string | undefined,
```